docs/decisions/0046-kernel-content-graduation.md

```diff
@@ -85,7 +85,7 @@ Pros:
 - With no deferred content we have simpler API and a single responsibility for contents.
 - Can be written and read in both `Data` or `DataUri` formats.
 - Can have a `Uri` reference property, which is common for specialized contexts.
-- Fully serializeable.
+- Fully serializable.
```

Comment:
Unrelated change. The spell checker's dictionary must have changed, since this was passing before, but is now failing the build.